Assets/Mirage/Runtime/CharacterSpawner.cs

```diff
@@ -53,7 +53,7 @@ public enum PlayerSpawnMethod { Random, RoundRobin }
 
 
         // Start is called before the first frame update
-        public virtual void Awake()
+        protected virtual void Awake()
```

Comment:
```suggestion
        protected internal virtual void Awake()
```
awake is called from tests so needs to be internal